client/src/components/chat/Chat.js

```diff
@@ -31,6 +31,10 @@ import ChatForm from '../forms/ChatForm';
 import ChatMessage from './ChatMessage';
 
 
+import chatRequester from '../../mock_data/conversation_action.json';
+import chatApprover from '../../mock_data/conversation_action.1.json';
```

Comment:
These imports have been removed. They should no longer be in `Chat.js`. Indication of outstandin merge issue.